treeherder/model/migrations/0055_autoclassify_status.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('model', '0054_duplicate_failure_classifications'),
+    ]
+
+    operations = [migrations.RunSQL(
+        sql="""
+BEGIN;
+--
+-- Add field autoclassify_status to job
+--
+SET FOREIGN_KEY_CHECKS=0;
+ALTER TABLE `job` ADD COLUMN `autoclassify_status` integer DEFAULT 0 NOT NULL;
+ALTER TABLE `job` ALTER COLUMN `autoclassify_status` DROP DEFAULT;
+COMMIT;
```

Comment:
Only if the `set foreign_key_checks` thing is a good idea in this case. Maybe it's not since no foreign keys are affected.